src/xdocs/config_javadoc.xml

```diff
@@ -3600,6 +3608,38 @@ public int foobar() {
               <td>6.2</td>
             </tr>
           </table>
+          <br/>
+          <p>
+            As per <a href="https://docs.oracle.com/javase/10/docs/specs/doc-comment-spec.html#inline-tags">
+            Documentation Comment Specification for the Standard Doclet</a> allowed tags inside
+            inline summary javadoc are
+          </p>
+          <table>
```

Comment:
Please do not use table, inline list of tags to above paragraph

Reply:
Done